config/linux-kgpe-d16_server.config

```diff
@@ -0,0 +1,326 @@
+CONFIG_LOCALVERSION="-heads"
```

Comment:
Must delete

Reply:
@tlaurion delete in `linux-kgpe-d16_server-whiptail.config` and `linux-kgpe-d16_workstation.config` also?

Reply:
> Must delete

this is in the Linux config, not coreboot, so it's still used if present. I don't think we ever discussed removing it from the kernel build string

Reply:
Absolutely right. Will review only with coffee from now on.